crates/re_space_view_time_series/src/space_view_class.rs

```diff
@@ -164,6 +165,15 @@ It can greatly improve performance (and readability) in such situations as it pr
             indicated_entities.0.extend(indicated.0.iter().cloned());
         }
 
+        // Because SeriesLine is our fallback visualizer, also include any entities for which
+        // SeriesLine is applicable, even if not indicated.
+        if let Some(applicable) = ctx
+            .applicable_entities_per_visualizer
+            .get(&SeriesLineSystem::identifier())
+        {
+            indicated_entities.0.extend(applicable.iter().cloned());
+        }
```

Comment:
ah great catch! At first I thought a different mechanism needs to do this, but I think that's spot on!